src/main/java/org/broadinstitute/hellbender/utils/recalibration/covariates/ContextCovariate.java

```diff
@@ -38,8 +38,6 @@ public ContextCovariate(final RecalibrationArgumentCollection RAC){
         mismatchesContextSize = RAC.MISMATCHES_CONTEXT_SIZE;
         indelsContextSize = RAC.INDELS_CONTEXT_SIZE;
 
-        logger.info("\t\tContext sizes: base substitution model " + mismatchesContextSize + ", indel substitution model " + indelsContextSize);
-
```

Comment:
I've restored it for you, but I have the bad feeling that it's going to be spamming my log again.